Makefile

```diff
@@ -12,4 +13,14 @@ push/image:
 	@echo "Pushing image u-vocab-api:${VERSION}"
 	@echo " "
 	docker login -u="${DOCKER_LOGIN}" -p="${DOCKER_PASSWORD}"
-	docker push uvocab/u-vocab-api:${VERSION}
\ No newline at end of file
+	docker push uvocab/u-vocab-api:${VERSION}
+
+deploy/image:
+	@echo " "
+	@echo "Deploy image u-vocab-api:${VERSION}"
+	@echo " "
+	ssh -i "${SSH_KEY}" "${EC2_USER}"@"${EC2_INSTANCE}" docker run --name u-vocab-api uvocab/u-vocab-api:${VERSION}
```

Comment:
@Danilosimoes my suggesstion is put the cmd commands in other file like `deploy.sh` and run it via ssh, why ? because is more flexible, for exemple, before run a container we need check if already exist other running... here a example we used in ontracker:
```
#!/bin/bash

DOCKER_LOGIN=$1
DOCKER_PASSWORD=$2
VERSION=$3


sudo systemctl enable docker
docker login -u $DOCKER_LOGIN -p $DOCKER_PASSWORD
docker rm -f $(docker ps -aq --filter="name=ontracker-api")
docker pull ontracker/ontracker-api:$VERSION
docker run --restart=always -d \
--log-opt max-size=10m --log-opt max-file=1 \
-e TZ=America/Sao_Paulo \
--name ontracker-api \
-p 8090:8090 -p 1099:1099 \
-m=950m \
ontracker/ontracker-api:$VERSION
```

`observe`, in docker run command we need pass more argument like the ports, size of memory...